### PR TITLE
fix

### PR DESCRIPTION
**Pull request by anri-1151**

### CODE REVIEW

File: app/autoupdater.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+'use strict';
+const os = require('os');
+const { app, autoUpdater, dialog } = require('electron');
+const version = app.getVersion();
+const platform = os.platform() + '_' + os.arch();  // usually returns darwin_64
+const config = require('./config');
+const updaterServerUrl = config.updateEndpoint;
+
+const updaterFeedURL = `${updaterServerUrl}${platform}/${version}`;
```
Comment (dfcowell) [inline app/autoupdater.js]: I need to give you the full URL for our feed @anri-1151 - I'll do that by tomorrow.

**Comment by dfcowell** [comment]
@anri-1151 here's the example usage for our update server:

```
import { app, autoUpdater } from 'electron';

const DOMAIN = 'https://selfkey-releases.storage.googleapis.com';
const suffix = process.platform === 'darwin' ? `/RELEASES.json?method=JSON&version=${app.getVersion()}` : '';
autoUpdater.setFeedURL({
  url: `${DOMAIN}/SelfKey-Desktop-Identity-Wallet/ea788838ccc9eb63676dd2da87772850/${process.platform}/${process.arch}${suffix}`,
  serverType: 'json',
});
```

**Comment by dfcowell** [comment]
> Also I think it would be better UX if the user was able to restart the app immediately rather than deferring it to next time

@altninja there's an "install and relaunch" button in the dialog box.

**Comment by altninja** [comment]
@dfcowell ok thats great, wasn't clear from the message text content so all good then

**Comment by dfcowell** [comment]
@altninja are you going to use this branch when you implement automatic updates, or should I close it?

**Comment by altninja** [comment]
merged this branch into standards PR - can close

**Pull request merged**